raincloudy/core.py

```diff
@@ -39,10 +40,10 @@ def __init__(self, username, password, http_proxy=None, https_proxy=None,
         self._password = password
 
         # initialize future attributes
-        self.controllers = []
+        self._controllers = []
         self.client = None
         self.is_connected = False
-        self.html = {
+        self._html = {
```

Comment:
making this protected so we throw linting errors for any children that try to access. `html` is no longer global since different controllers need slightly different page loads